src/NServiceBus.Gateway.Tests/APIApprovals.Approve.approved.txt

```diff
@@ -13,7 +13,7 @@ namespace NServiceBus
     public class GatewaySettings
     {
         public void ChannelFactories(System.Func<string, NServiceBus.Gateway.IChannelSender> senderFactory, System.Func<string, NServiceBus.Gateway.IChannelReceiver> receiverFactory) { }
-        public void CustomRetryPolicy(System.Func<NServiceBus.Transports.IncomingMessage, System.Exception, int, System.TimeSpan> customRetryPolicy) { }
+        public void CustomRetryPolicy(System.Func<NServiceBus.Transport.IncomingMessage, System.Exception, int, System.TimeSpan> customRetryPolicy) { }
```

Comment:
Sorry didn't catch this in the first round. Should we call this CustomDelayedRetryPolicy since we only allow to customize the delayed part